shotover-proxy/src/transforms/cassandra/sink_cluster/routing_key.rs

```diff
@@ -54,7 +56,7 @@ fn serialize_routing_key_with_indexes(
 }
 
 // https://github.com/apache/cassandra/blob/3a950b45c321e051a9744721408760c568c05617/src/java/org/apache/cassandra/db/marshal/CompositeType.java#L39
-fn serialize_routing_value(cursor: &mut Cursor<&mut Vec<u8>>, value: &Value, version: Version) {
+fn serialize_routing_value(cursor: &mut Cursor<&mut Vec<u8>>, value: &Vec<u8>, version: Version) {
     let temp_size: CIntShort = 0;
```

Comment:
We can now change this to `let size: CIntShort = value.len()`, allowing us to remove the:
```rust
    let after_value_pos = cursor.position();
    cursor.set_position(before_value_pos - SHORT_LEN as u64);
    let value_size: CIntShort = (after_value_pos - before_value_pos) as CIntShort;
    value_size.serialize(cursor, version);
```